signer/src/storage/model.rs

```diff
@@ -214,7 +214,7 @@ impl From<Deposit> for DepositRequest {
         // It's most likely the case that each of the inputs "came" from
         // the same Address, so we filter out duplicates.
         let sender_script_pub_keys: BTreeSet<ScriptPubKey> = tx_input_iter
-            .map(|tx_in| tx_in.prevout.script_pub_key.script.into())
+            .filter_map(|x| Some(x.prevout?.script_pub_key.script.into()))
```

Comment:
For coinbase, we end up with an empty `sender_script_pub_keys`. I guess that's correct, and since we use `sender_script_pub_keys` only for the blocklist check, this is fine. I just wanted to double check this is what we want -- i.e., if we lift the deposit coinbase check, we would (correctly) create the request with empty `sender_script_pub_keys`.

Reply:
I think this is what we want. We need input addresses but don't have any, so we should be fine. I'm not sure what else we could do.